azkaban-common/src/main/java/azkaban/metrics/CommonMetrics.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.metrics;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
+
+/**
+ * This singleton class CommonMetrics is in charge of collecting varieties of metrics
+ * from azkaban-common modules.
+ */
+public enum CommonMetrics {
+  INSTANCE;
+
+  private Meter _dbConnectionMeter;
+  private MetricRegistry _metrics;
+
+  CommonMetrics() {
+    _metrics = MetricsManager.INSTANCE.getRegistry();
```

Comment:
I am a bit concerned about the coding style here.
Azkaban uses a different coding style than LinkedIn. If we are moving to LinkedIn standard, let's change the entire codebase and move to it. Else let's stick to Azkaban's default. Having both styles in the same codebase may not be the best idea